<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesOrPurchaseOrder</class>
 <widget class="QDialog" name="SalesOrPurchaseOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="text">
      <string>Sales order</string>
     </property>
     <property name="description">
      <string>Make a record of sales order</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="text">
      <string>Purchase order</string>
     </property>
     <property name="description">
      <string>Make a record of purchase order</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="commandLinkButton_3">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
